modules/rosa-sts-aws-requirements.adoc

```diff
@@ -138,4 +143,4 @@ If you do not specify a profile, the default AWS profile is used.
 == Security requirements
 * Volume snapshots will remain within the customer's AWS account and customer-specified region.
 * Red Hat must have ingress access to EC2 hosts and the API server from allow-listed IP addresses.
-* Red Hat must have egress allowed to the documented domains.
+* Red Hat must have egress allowed to the documented domains. See the "AWS firewall prerequisites" section for the designated domains.
```

Comment:
If you split the module as suggested, you could even do `.Additional resources` links to sections farther down the page (I have had to do this for similar situations)